docs/pages/example/hillshade.html

```diff
@@ -0,0 +1,31 @@
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/outdoors-v9',
+    center: [-119.5857, 37.7221],
+    zoom: 11
+});
+
+map.on('load', function () {
+    map.removeLayer('hillshade_shadow_extreme');
+    map.removeLayer('hillshade_shadow_dark');
+    map.removeLayer('hillshade_shadow_med');
+    map.removeLayer('hillshade_shadow_faint');
+    map.removeLayer('hillshade_highlight_med');
+    map.removeLayer('hillshade_highlight_bright');
```

Comment:
@mollymerp This was one thing I'm not sure about. In terms of an example for the hillshade layer, it might make better sense to just use something like the basic style and add hillshading to that. It would be a simpler example.

This example here is more complex due to it removing those existing hillshade layers from outdoors client side. Especially given any real map wanting Outdoors with raster hill shading could first cut the Hillshade layers from Outdoors in Studio as a new style rather than doing it client side... :thought_balloon:

Reply:
yeah good point – I made a new public style in the Mapbox account that is Outdoors minus the hillshade layers,so you can use this style id and explain with a comment what it is ?

mapbox://styles/mapbox/cjaudgl840gn32rnrepcb9b9g

Reply:
Great. Assuming you're okay that people will copy the example and rely on that style not disappearing in the future?

Reply:
yep – and we can replace it with a permanent style whenever our core styles start using this layer type.